packages/atomic/src/components/atomic-smart-snippet/atomic-smart-snippet-feedback-banner.tsx

```diff
@@ -0,0 +1,105 @@
+import {h, FunctionalComponent} from '@stencil/core';
+import Checkmark from '../../images/checkmark.svg';
+import Cross from '../../images/cross.svg';
+import {i18n} from 'i18next';
+import {RadioButton} from '../common/radio-button';
+
+interface SmartSnippetFeedbackBannerProps {
+  i18n: i18n;
+  id: string;
+  liked: boolean;
+  disliked: boolean;
+  onLike(): void;
+  onDislike(): void;
+}
+
+export const SmartSnippetFeedbackBanner: FunctionalComponent<
+  SmartSnippetFeedbackBannerProps
+> = (props) => {
+  const inquiryId = 'feedback-inquiry-' + props.id;
+  const thankYouId = 'feedback-thank-you-' + props.id;
+  const radioGroupName = 'feedback-options-' + props.id;
```

Comment:
I decided to make this a functional component since it doesn't really have a state of its own, and unlike the expandable answer, doesn't need to render multiple times.

However, a side effect of this, is that `randomID()` would be executed multiple times and would keep changing the ID & group name of various components, which I fear could affect screen readers. My solution was to take a `id` as a prop.